website_sale_product_brand/static/src/js/tour.js

```diff
@@ -0,0 +1,43 @@
+/* License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+ odoo.define("website_sale_product_brand.tour", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var tour = require("web_tour.tour");
+
+    var _t = core._t;
+
+    var steps = [
+        {
+            trigger: "a[href='/page/product_brands']",
+            content: _t("Go to 'Product brand' page"),
```

Comment:
If this tour is only for testing (not for guiding first-time users), then don't translate strings.